Jenkinsfile

```diff
@@ -81,7 +81,7 @@ pipeline {
                 }
             }
             steps {
-                withMaven(maven: 'Maven 3.5.4', jdk: 'jdk11', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'cx-internal', mavenOpts: "${LINUX_MVN_RANDOM}") {
+                withMaven(maven: 'Maven 3.5.4', jdk: 'jdk11', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'ion-maven-settings', mavenOpts: "${LINUX_MVN_RANDOM}") {
```

Comment:
❓Will an `ion-maven-settings` config file have to be added to the ION space in Jenkins for the build to work? Or will it be able to grab it from the global managed files?

Reply:
I would hope that config already exists since it is used by the ingest api. https://github.com/connexta/ion-ingest-api/blob/52b3078ff430783b8fce23fd7d61cf58040d107d/Jenkinsfile#L82